### ISSUE TITLE
[Feature request] Add the app to winget & chocolatey

### ISSUE THREAD

**Issue by Twig6943** [amanharwara/altus#336]
I'm on linux (I dont use windows) but this should help windows users
(Not sure if this is already a thing)

**Comment by philipmgrant** [comment]
It's already on Altus (package ID `AmanHarwara.Altus`).

Not on Chocolatey yet, as far as I can tell.

**Comment by Noellelivinggayness** [comment]
yeah I used winget I love it because I can update my things in one command although I have to tweak things to have it install discretely but I almost installed the executable because there's no instruction on readme that says where it's available
